### PR TITLE
Rework memdump class

### PR DESCRIPTION
**Pull request by pylessard**

### CODE REVIEW

File: python/scrutiny/server/device/request_generator/device_searcher.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class DeviceSearcher:
     DISCOVER_INTERVAL = 0.5
     DEVICE_GONE_DELAY = 3
 
-    def __init__(self, protocol, dispatcher, priority=10):
```
Comment (fredericmorin) [inline python/scrutiny/server/device/request_generator/device_searcher.py:21]: Magic value. Use a constant here. Or something like "LOWER_PRIORITY" ou "HIGHEST_PRIORITY"

File: python/test/cli/test_makevarmap_Linux_LE_x64_Gcc_8_3_0.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 from scrutiny.core import *
 from scrutiny.core.bintools.elf_dwarf_var_extractor import ElfDwarfVarExtractor
-from scrutiny.core.memdump import Memdump
+from scrutiny.core.memcontent import MemContent
```
Comment (fredericmorin) [inline python/test/cli/test_makevarmap_Linux_LE_x64_Gcc_8_3_0.py:17]: Careful about abbreviations for class names. `MemoryContent` would be a better name IMHO

File: python/test/core/test_memcontent.py [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+        memcontent.write(0x1000, data)
+        memcontent.write(0x2000, data)
+        memcontent.write(0x3000, data)
+
+        return memcontent
+
+    def test_delete_single_cluster_middle(self):
+        import ipdb
+        m = self.delete_test_make_mem()
+        # ipdb.set_trace()
```
Comment (fredericmorin) [inline python/test/core/test_memcontent.py:187]: Cleanup developper support imports statement `import ipdb` and ipdb related comments.

**Review by fredericmorin — APPROVED** [review]

File: python/test/core/test_memcontent.py (hunk 3 above)
Comment (pylessard) [reply python/test/core/test_memcontent.py]: Fixed

File: python/test/cli/test_makevarmap_Linux_LE_x64_Gcc_8_3_0.py (hunk 2 above)
Comment (pylessard) [reply python/test/cli/test_makevarmap_Linux_LE_x64_Gcc_8_3_0.py]: MemoryContent it is

File: python/scrutiny/server/device/request_generator/device_searcher.py (hunk 1 above)
Comment (pylessard) [reply python/scrutiny/server/device/request_generator/device_searcher.py:21]: removed default priority instead.

**Pull request merged**